erdpy/dependencies/modules.py

```diff
@@ -245,6 +245,41 @@ def get_latest_release(self) -> str:
         raise errors.UnsupportedConfigurationValue("Nodejs tag must always be explicit, not latest")
 
 
+class NpmModule(DependencyModule):
+    def __init__(self, key: str, aliases: List[str] = []):
+        super().__init__(key, aliases)
+
+    def get_nodejs(self) -> DependencyModule:
+        return dependencies.get_module_by_key("nodejs")
+    
+    def get_nodejs_env(self) -> Dict[str, str]:
+        return self.get_nodejs().get_env()
+
+    def _do_install(self, tag: str) -> None:
```

Comment:
`wasm-opt` is installed globally (`-g`), but under `elrondsdk / (...) / node_modules`, wrt. `get_nodejs_env()`. It's all right :+1: